store/prefix/store_test.go

```diff
@@ -1,17 +1,28 @@
-package store
+package prefix
 
 import (
 	"math/rand"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
-	"github.com/tendermint/iavl"
+	tiavl "github.com/tendermint/iavl"
 	dbm "github.com/tendermint/tendermint/libs/db"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/store/dbadapter"
+	"github.com/cosmos/cosmos-sdk/store/iavl"
+	"github.com/cosmos/cosmos-sdk/store/types"
 )
 
+// copied from iavl/store_test.go
+var (
+	cacheSize        = 100
```

Comment:
These should really be config parameters

Reply:
++

Reply:
Still true - could be a separate PR though.